patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -58,17 +58,27 @@
          return p_77621_1_.func_147447_a(vec3d, vec3d1, p_77621_3_, !p_77621_3_, false);
      }
  
-@@ -409,6 +412,9 @@
+@@ -409,8 +412,18 @@
  
      protected boolean func_194125_a(CreativeTabs p_194125_1_)
      {
 +        for (CreativeTabs tab : this.getCreativeTabs())
 +            if (tab == p_194125_1_)
 +                return true;
++
          CreativeTabs creativetabs = this.func_77640_w();
-         return creativetabs != null && (p_194125_1_ == CreativeTabs.field_78027_g || p_194125_1_ == creativetabs);
+-        return creativetabs != null && (p_194125_1_ == CreativeTabs.field_78027_g || p_194125_1_ == creativetabs);
++        if (creativetabs != null) {
```

Comment:
{ mising newline

Reply:
Patches don't follow the same style as Forge classes. They are supposed to be minimal, so braces don't get a new line.